crowdin_api/api_resources/source_files/resource.py

```diff
@@ -421,7 +424,7 @@ def delete_file(self, projectId: int, fileId: int):
             path=self.get_file_path(projectId=projectId, fileId=fileId),
         )
 
-    def edit_file(self, projectId: int, fileId: int, data: Iterable[FilePatchRequest]):
+    def edit_file(self, projectId: int, context: str, fileId: int, data: Iterable[FilePatchRequest]):
```

Comment:
we should only update the `FilePatchRequest` here, the `edit_file` parameters shouldn't be chnged